vignettes/01_rphenoscape.Rmd

```diff
@@ -74,15 +74,15 @@ find_term("pectoral fin", matchTypes = c("exact"))
 For a specific taxon, such as Siluriformes, you can view the taxon rank, term label, whether or not the taxon is extinct, its common name, and ID: 
 
 ```{r}
-pk_taxon_detail("Siluriformes")
+taxon_info("Siluriformes")
 ```
 
-Ontology terms are related to each other by superclass (parent) and subclass (child) relationships, among other relationships such as part_of. You can use the term classification functions ```pk_anatomical_class()```, ```pk_taxon_class()```, and ```pk_phenotype_class()```to view the superclass and subclass relationships for anatomy, taxonomy, or phenotypic quality terms. 
+Ontology terms are related to each other by superclass (parent) and subclass (child) relationships, among other relationships such as part_of. You can use ```term_classification()```to view the superclass and subclass relationships for a given term (e.g., anatomy, taxon).
 
 For example, view the relationships for “pectoral fin”: 
 
 ```{r}
-pk_anatomical_class("pectoral fin")
+term_classification("pectoral fin", as = "anatomy")
```

Comment:
Using `as.terminfo()`, simply printing the object to the terminal will surface this information. And it's also accessible programmatically from the object returned by `as.terminfo` (use `names(terminfo_object)` to find out which elements the object has).

Reply:
done